### PR TITLE
Update boilerplate on entry index page for NOIRLab rather than NOAO

### PR DESCRIPTION
**Pull request by geordie666**
NOIRLab reached out asking if we could update the boilerplate on the main page to reflect NOIRLab-appropriate language instead of the older NOAO-appropriate language.

This PR makes that change.

### CODE REVIEW

**Comment by geordie666** [comment]
@weaverba137: This is a minor update requested by a NOIRLab Public Information Officer. I don't think this is particularly urgent, but please merge this PR when you have time to serve a new version of the website.

**Comment by weaverba137** [comment]
That's fine, just wanted to double-check. Merging shortly.

File: pages/index.rst [hunk 1]
```diff
@@ -68,10 +68,17 @@
...
+   Laboratory), the US center for ground-based optical astronomy,
+   operates the international Gemini Observatory (a facility of NSF
+   NRC-Canada, ANID-Chile, MCTIC-Brazil, MINCyT-Argentina, and
+   KASI-Republic of Korea), Kitt Peak National Observatory (KPNO),
+   Cerro Tololo Inter-American Observatory (CTIO), the Community
+   Science and Data Center (CSDC), and Vera C. Rubin Observatory (in
+   cooperation with DOE's SLAC National Accelerator Laboratory). It is
+   managed by the Association of Universities for Research in Astronomy
+   (AURA) under a cooperative agreement with NSF and is headquartered
+   in Tucson, Arizona*
```
Comment (weaverba137) [inline pages/index.rst:81]: I assume there should be a period at the end of this sentence?

**Review by weaverba137 — APPROVED** [review]
I can fix the punctuation if there are no other changes planned.

File: pages/index.rst (hunk 1 above)
Comment (geordie666) [reply pages/index.rst:81]: Oops, sorry, yes, thanks. If you want to check against the actual language I was sent, here's a copy/paste:

```
[NSF’s NOIRLab](https://noirlab.edu/) (National Optical-Infrared Astronomy Research Laboratory), the US center for ground-based optical-infrared astronomy, operates the international [Gemini Observatory](https://www.noirlab.edu/public/programs/gemini-observatory/) (a facility of [NSF](https://www.nsf.gov/), [NRC–Canada](http://www.nrc-cnrc.gc.ca/eng/solutions/facilities/gemini.html), [ANID–Chile](http://www.conicyt.cl/astronomia/oficina-gemini-chile/), [MCTIC–Brazil](https://www.gov.br/mcti/pt-br), [MINCyT–Argentina](http://www.geminiargentina.mincyt.gob.ar/), and [KASI–Republic of Korea](http://kgmt.kasi.re.kr/kgmtscience)), Kitt Peak National Observatory ([KPNO](https://www.noirlab.edu/public/programs/kitt-peak-national-observatory/)), Cerro Tololo Inter-American Observatory ([CTIO](https://www.noirlab.edu/public/programs/ctio/)), the Community Science and Data Center ([CSDC](https://www.noirlab.edu/public/programs/csdc/)), and [Vera C. Rubin Observatory](https://www.noirlab.edu/public/programs/vera-c-rubin-observatory/) (in cooperation with [DOE](https://www.energy.gov/science/office-science)’s [SLAC](https://www6.slac.stanford.edu/) National Accelerator Laboratory). It is managed by the Association of Universities for Research in Astronomy ([AURA](https://www.aura-astronomy.org/)) under a cooperative agreement with [NSF](https://www.nsf.gov/) and is headquartered in Tucson, Arizona.
```

I decided to drop the urls as links aren't included elsewhere in the boilerplate and to make maintenance easier.

Comment (geordie666) [reply pages/index.rst:81]: (I have no other planned changes).

Comment (weaverba137) [reply pages/index.rst:81]: Yikes, I found another difference: "ground-based optical-infrared astronomy" somehow got changed to "ground-based optical astronomy". I'm going go through word-for-word.

**Comment by weaverba137** [comment]
I'll merge after the latest round of tests pass.

**Comment by geordie666** [comment]
Thanks — apologies for mangling some of the text when I removed the links. I should definitely have checked that more carefully.

**Pull request merged**